var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -186,10 +186,10 @@ def configure_args(self):
             "--with-blas-lib-dirs=%s" % " ".join(blas.directories),
         ]
 
-        if "+mpi" in spec:
+        if spec.satisfies("^mpi"):
```

Comment:
Are you checking to see if the `mpi` variant is enabled?  If so, the string argument is the same as it was before (i.e., `spec.satisfies("+mpi")).

Reply:
Gotcha, I got confused with variant / dependency syntax. Thanks for catching!